code/__defines/gamemode.dm

```diff
@@ -25,6 +25,8 @@
 #define BE_MUTINEER   0x2000
 #define BE_PAI        0x4000
 #define BE_LOYALIST   0x8000
+#define BE_HACKER     0x10000
+#define BE_SPY        0x20000
```

Comment:
Sorry, forgot one: You cannot do this. These would be bits 17 and 18 in a 16-bit number.

Reply:
I feel like you would need to make these two the same pref, and even then it wouldn't have enough room.

Reply:
Oof. I didn't know how the '0xnumber' things even worked, or why they existed. What's your suggestion for the flags, then?

Reply:
Not sure. Are all those prefs still needed? Can you remove any?

Reply:
`0x` is a hexadecimal prefix, so any number following it is interpreted as hexadecimal. `0b`, similarly, is a binary prefix.
`0x10 == `0b00010000`
`0xFF == 0b11111111`

Reply:
Don't know, but that's the standard in instances where it is supported.